gears/flywheel/freesurferator.json

```diff
@@ -0,0 +1,225 @@
+{
+  "name": "freesurferator",
+  "label": "freesurferator: create anatomical ROIs for DWI, PET or fMRI in subject space.",
+  "description": "This gear takes either an existing FS zipped run, or an anatomical NIfTI file and performs all of the FreeSurfer cortical reconstruction process. Outputs are provided in a zip file and include the entire output directory tree from Recon-All. Configuration options exist for setting the subject ID and for converting outputs to NIfTI, OBJ, and CSV. FreeSurfer is a software package for the analysis and visualization of structural and functional neuroimaging data from cross-sectional or longitudinal studies. It is developed by the Laboratory for Computational Neuroimaging at the Athinoula A. Martinos Center for Biomedical Imaging. Please see https://surfer.nmr.mgh.harvard.edu/fswiki/FreeSurferSoftwareLicense for license information. This gear includes a bunch of atlases included (cerebellum, HCP, Neuropythy... and depending on the configuration options it will create many ROIs in subject space that can later be incorporated to other neuroimaging tools (we use it extensively with DWI data, namely RTP-pipeline",
```

Comment:
@garikoitz can you fix the parenthesis typo toward the end. 
Also rephrase "This gear includes a bunch of atlases included"